newswires/app/db/QueryParams.scala

```diff
@@ -0,0 +1,19 @@
+package db
+
+import conf.SearchTerm
+
+sealed trait UpdateType {
+  val sinceId: Int
+}
+
+case class NextPage(sinceId: Int) extends UpdateType
+case class MostRecent(sinceId: Int) extends UpdateType
```

Comment:
`MostRecent` is not currently used, but I am planning to use it in an upcoming PR. Can remove it here though if we aren't happy with adding the abstraction prematurely.